doc_source/DNSLimitations.md

```diff
@@ -83,7 +83,7 @@ For information on getting current quotas \(formerly referred to as "limits"\),
 | Associations between rules and VPCs per AWS Region | 2000 per AWS account [Request a higher quota](https://console.aws.amazon.com/servicequotas/home?region=us-east-1#!/services/route53/quotas)\.  | 
 | Queries per second per IP address in an endpoint | 10,000\* | 
 
-**\*** The number of DNS queries per second varies by the type of query, the size of response, and the protocol in use\. For information about how to view Amazon CloudWatch metrics for IP addresses for inbound and outbound endpoints, see [Metrics for Resolver IP addresses](monitoring-resolver-with-cloudwatch.md#cloudwatch-metrics-resolver-ip-address)\.
+**\*** The number of DNS queries per second varies by the type of query, the size of response, and the protocol in use\. For information about how to view Amazon CloudWatch metrics for IP addresses for inbound and outbound endpoints, see [Metrics for Resolver IP addresses](monitoring-resolver-with-cloudwatch.md#cloudwatch-metrics-resolver-ip-address)\. Also, this limit of 10,000 QPS is on a per elastic network interface basis. For example, 2 ENIs, which would give 20,000 qps. From there you can scale out by adding more ENIs.
```

Comment:
Before using QPS as an acronym, we should define what it means. I also think we should clarify where the ENI acronym comes from before using it. And QPS should always be capitalized. So your first couple of added sentences should say:

> Also, this limit of 10,000 queries per second (QPS) is on a per elastic network interface (ENI) basis. For example, 2 ENIs, which would give 20,000 QPS.

BTW, I love that you added an example after that first sentence!

Reply:
Hi @joshbean Thanks for the reply. I agree with your comments about expanding the abbreviations QPS and ENI. I overlooked in my initial request. If you are OK with the text then please go ahead and merge the change as you see fit.

Reply:
Will do!